src/hotspot/share/gc/shenandoah/shenandoahEvacInfo.hpp

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (c) 2013, 2021, Oracle and/or its affiliates. All rights reserved.
```

Comment:
For these files introduced by us, we should put an Amazon copyright notice.  See, for example, shenandoahGenerationallHeuristics.cpp.

Reply:
Thanks for fixing this